From cd40339ce00ea4b85dc564c59e8e37f43f489369 Mon Sep 17 00:00:00 2001
From: Gao Pan <pandy.gao@nxp.com>
Date: Wed, 8 Mar 2017 22:37:23 +0800
Subject: [PATCH 1575/5242] MLK-14394 i2c: imx-lpi2c: change default bitrate

commit  2ca9d3423a86559291a3df6120d372e0c510e1e6 from
https://source.codeaurora.org/external/imx/linux-imx.git

Due to the eviation i2c clk settings, the result may less than
100KBps. As a result, some i2c slave works in wrong condition.

This patch changes i2c bitrate to 200KBps

Signed-off-by: Gao Pan <pandy.gao@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/i2c/busses/i2c-imx-lpi2c.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/i2c/busses/i2c-imx-lpi2c.c b/drivers/i2c/busses/i2c-imx-lpi2c.c
index f6d1183..85cc2e4 100644
--- a/drivers/i2c/busses/i2c-imx-lpi2c.c
+++ b/drivers/i2c/busses/i2c-imx-lpi2c.c
@@ -75,7 +75,7 @@
 #define I2C_CLK_RATIO	2
 #define CHUNK_DATA	256
 
-#define LPI2C_DEFAULT_RATE	100000
+#define LPI2C_DEFAULT_RATE	200000
 #define STARDARD_MAX_BITRATE	400000
 #define FAST_MAX_BITRATE	1000000
 #define FAST_PLUS_MAX_BITRATE	3400000
-- 
1.7.9.5

